src/index.js

```diff
@@ -12,7 +12,8 @@ import MLDivider from './MLDivider'
 import * as MLIcon from './MLIcon'
 import MLInputNumber from './MLInputNumber'
 import MLLayout from './MLLayout'
-import MLMentions from './MLMentions'
```

Comment:
I'm guessing this erroneous renaming is from rebasing incorrectly. Gotta be careful with rebasing; it's a lost easier to get lost in than with just one set of merge conflicts.